public/src/components/calls/side_bar_form.jsx

```diff
@@ -4,23 +4,20 @@ import { Field } from 'redux-form';
 
 import FieldGroup from '../../components/common/form/field_group';
 
-const SideBarForm = (props) => {
-  const { handleSubmit } = props;
+const SideBarForm = () => {
   const inputFieldProps = {
     label: 'Notes',
     name: 'notes'
   };
   return (
-    <div>
-      <form onSubmit={handleSubmit}>
```

Comment:
onSubmit/handleSubmit is unnecessary here, that's why I removed it

Reply:
👍